src/ios/removeFromStaticLibraries.js

```diff
@@ -14,6 +15,8 @@ module.exports = function removeFromStaticLibraries(project, path, opts) {
   project.removeFromPbxBuildFileSection(file);
   project.removeFromPbxFrameworksBuildPhase(file);
   project.removeFromLibrarySearchPaths(file);
+  project.removeFromProductsPbxGroup(file);  // @todo this does not work but it should
```

Comment:
It's something I have to understand a bit deeper since technically speaking we have ~10 `Products` groups so this method is for a slightly different use-case (however it's named in a similar way)